### PR TITLE
[5.4] Refactor make:auth

### PR DESCRIPTION
**Pull request by srmklive**
This PR adds further features to `make:auth` command like:

* Set `views` & `force` options as optional values not default.
* Check whether `HomeController.php` already exists.
* Don't add auth routes again if they are already added.

### CODE REVIEW

File: src/Illuminate/Auth/Console/MakeAuthCommand.php [hunk 1]
```diff
@@ -39,6 +47,11 @@ class MakeAuthCommand extends Command
         'home.stub' => 'home.blade.php',
     ];
 
+    public function __construct()
+    {
+        parent::__construct();
+    }
```
Comment (ntzm) [inline src/Illuminate/Auth/Console/MakeAuthCommand.php]: This can be taken out

Comment (srmklive) [inline src/Illuminate/Auth/Console/MakeAuthCommand.php]: Done.

**Comment by taylorotwell** [comment]
I'm not adding anymore features to this command. Let's not ruin a simple thing.

**Comment by ntzm** [comment]
I think

> Check whether HomeController.php already exists.

Should be kept

**Pull request closed without merging**